recipes/texlive-core/build.sh

```diff
@@ -0,0 +1,52 @@
+#! /bin/bash
+
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig:$PREFIX/share/pkgconfig"
+
+mkdir -p tmp_build && pushd tmp_build
+  ../configure --prefix=$PREFIX \
+               --with-cairo=$PREFIX \
+               --datarootdir="$PREFIX/share/texlive" \
+               --disable-all-pkgs \
+               --disable-native-texlive-build \
+               --disable-ipc \
+               --disable-debug \
+               --disable-dependency-tracking \
+               --disable-mf \
+               --disable-pmp \
+               --disable-upmp \
+               --disable-aleph \
+               --disable-eptex \
+               --disable-euptex \
+               --disable-luatex \
+               --disable-luajittex \
+               --disable-uptex \
+               --enable-web2c \
+               --enable-silent-rules \
+               --enable-tex \
+               --enable-etex \
+               --enable-pdftex \
+               --enable-xetex \
+               --enable-web-progs \
+               --enable-texlive \
+               --enable-dvipdfm-x \
+               --with-icu=$PREFIX \
+               --with-mpfr=$PREFIX \
+               --with-gmp=$PREFIX \
+               --with-pixman=$PREFIX \
+               --with-freetype2=$PREFIX \
+               --with-libpng=$PREFIX \
+               --with-zlib=$PREFIX \
+               --x-libraries=$PREFIX/lib \
+               --x-includes=$PREFIX/include \
+               --without-system-harfbuzz \
+               --without-system-graphite2 \
+               --without-system-poppler
+
+  make
+  make check
```

Comment:
Ignore that the logs are in each library directory. The issue is during "Web2C 2015" test suite. Here is what I found in `$SRC_DIR/tmp_build/texk/web2c/test-suite.log`.

```
==================================
   Web2C 2015: ./test-suite.log
==================================

# TOTAL: 69
# PASS:  68
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: pdftexdir/ttf2afm
=======================

*** ttf2afm postV3.ttf
sed: RE error: illegal byte sequence
*** ttf2afm postV7.ttf

Warning: ttf2afm (file postV7.ttf): unsupported format (00070000) of `post' table, assuming 3.0

Warning: ttf2afm (file postV7.ttf): no names available in `post' table, print glyph names as indices
sed: RE error: illegal byte sequence

failed tests: postV3 postV7

```